### PR TITLE
BadHub.cz

### PR DESCRIPTION
**Pull request by WaterChick**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/badhub/manifest.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "server_name": "badhub",
+  "nice_name": "BadHub",
+  "direct_ip": "mc.badhub.cz",
+  "server_wildcards": [
+    "%.badhub.cz"
```
Comment (maaaathis) [inline minecraft_servers/badhub/manifest.json:6]: Doesn't look like a real wildcard
image

File: minecraft_servers/badhub/manifest.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  "supported_languages": [
+     "cz",
+     "sk"
+  ],
+  "social": {
+    "web": "https://www.badhub.cz/",
+    "web_shop": "https://vip.vagonbrei.eu/link/LAkWETe4mc",
+    "tiktok": "mc.badhub.cz",
+    "instagram": "badhub.cz",
+    "discord": "https://discord.badhub.cz"
```
Comment (maaaathis) [inline minecraft_servers/badhub/manifest.json:17]: Custom invite links are reserved for partners only; use a normal discord.gg/[CODE] link instead

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by WaterChick** [comment]
I will let know when the wildcard records are live

File: minecraft_servers/badhub/manifest.json (hunk 1 above)
Comment (DoJapHD) [reply minecraft_servers/badhub/manifest.json:6]: Still not a wildcard
https://i.imgur.com/PkUrfbL.png

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by WaterChick** [comment]
Just added it. Can you check it ?

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**